src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,63 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t06_homework() {
 
+
+	int N;
+	int no;
+	cin >> N;
+	string a;
+	string work;
+	set <string> slov;
+	no = 0;
+	
+	for (int i = 0; i < N; i++){
+	
+		
+		cin >> a;
+		slov.insert(a);
+	}
+
+
+	while (cin >> work)
+	{
+		bool in = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'in' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4721709)